docs/schema/validation.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "title": "Validation settings",
+  "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+  "type": "object",
+  "properties": {
+    "nav": {
+      "title": "Settings for validation of navigation",
+      "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+      "properties": {
+        "omitted_files": {
+          "type": "string",
```

Comment:
I think there's no need to set `type: string` when you set `enum` ☺️ All instances of settings where we define `enum` values don't have a `type `defined, so I suggest to drop it.